src/category_theory/instances/topological_spaces.lean

```diff
@@ -134,6 +134,15 @@ def map
 def map_iso {X Y : Top.{u}} (f g : X ⟶ Y) (h : f = g) : map f ≅ map g :=
 nat_iso.of_components (λ U, eq_to_iso (congr_fun (congr_arg _ (congr_arg _ h)) _) ) (by obviously)
 
-@[simp] def map_iso_id {X : Top.{u}} (h) : map_iso (𝟙 X) (𝟙 X) h = iso.refl (map _) := rfl
+@[simp] lemma map_iso_id {X : Top.{u}} (h) : map_iso (𝟙 X) (𝟙 X) h = iso.refl (map _) := rfl
+@[simp] lemma map_iso_id' {X Y : Top.{u}} (f : X ⟶ Y) : map_iso f f (refl _) = iso.refl (map _) := rfl
```

Comment:
Are there occasions where `map_iso_id` would trigger, but `map_iso_id'` would not?

Reply:
I've cleaned this up.